misk-actions/src/main/kotlin/misk/exceptions/Exceptions.kt

```diff
@@ -103,6 +104,9 @@ open class ClientClosedRequestException(message: String = "", cause: Throwable?
  * Base exception for when a server is acting as a gateway and cannot get a response in time.
  * The message is not exposed to the caller.
  */
```

Comment:
Probably want this comment for gateway timeout, and a different comment for this exception.